client/src/pages/Submission/Submission.tsx

```diff
@@ -37,6 +39,13 @@ const submissionData = [
 
 export default function Submission(): JSX.Element {
   const classes = useStyles();
+  const [submission, setSubmission] = useState(submissionData);
+  const { contestId } = useParams<{ contestId: string }>();
+
+  useEffect(() => {
+    if (contestId) {
```

Comment:
Please remove?

Reply:
Is it the use Effect. I put it there so I could get the url contestID parameter when there is change in contestID

Reply:
Then query the database for that contest

Reply:
You can add later when you need the hook. However, try not to leave partial code in PRs as we may later forget to remove/edit